src/v2/Components/NavBar/Menus/DropDownMenu/DropDownMenu.tsx

```diff
@@ -42,58 +65,36 @@ export const DropDownNavMenu: React.FC<DropDownNavMenuProps> = ({
     }
   }
 
+  const viewAllTopMargin = VIEW_ALL_TOP_MARGINS[contextModule]
+
   return (
     <Menu onClick={handleClick} width={width} py={0}>
       <Flex justifyContent="center">
-        <SimpleLinksContainer
-          py={3}
-          mr={[3, 3, 3, "50px"]}
-          viewAllTopMargin={viewAllTopMargin[contextModule]}
-        >
+        <Links py={3} mr={[3, 3, 3, "50px"]}>
           <Box mr={[1, 1, 2, 2]} width={[110, 110, 110, 135, 170]}>
-            {menu.links.map(menuItem => {
-              if (!menuItem.menu) {
-                return (
-                  <MenuItemContainer key={menuItem.text}>
-                    <MenuItem
-                      px={1}
-                      py={0.5}
-                      href={menuItem.href}
-                      textColor={color("black60")}
-                      textWeight="regular"
-                      fontSize="3t"
-                      hasLighterTextColor
-                    >
-                      {menuItem.text}
-                    </MenuItem>
-                  </MenuItemContainer>
+            {menu.links.map((menuItem, i) => {
+              if (!("menu" in menuItem)) {
```

Comment:
I noticed this in a few places:

`menuItem.menu` guards in JavaScript but isn't a type guard. `"menu" in menuItem` [will discriminate the union for you.](https://www.typescriptlang.org/docs/handbook/advanced-types.html#using-the-in-operator)

Reply:
ah cool!